frontend/src/components/experimentRuns/ExperimentRunValidation.tsx

```diff
@@ -59,53 +59,36 @@ function ExperimentRunValidation({ experimentRunName }: ExperimentRunValidationP
 
 	}, [experimentRunName, experimentRunLanesState])
 
+	const updateLane = useCallback((lane: LaneInfo) => {
+		Promise.allSettled(lane.datasets.map((dataset) => dispatch(get(dataset.datasetID)))).finally(() => {
```

Comment:
Any clue why this is working without await ? I spent a good while without it having the promise returning right away ... From looking at the console you need the await otherwise you get spammed with calls to the backend ... it ends up falling back on its feet but i think it is a bit random.

Reply:
> From looking at the console you need the await otherwise you get spammed with calls to the backend ... it ends up falling back on its feet but i think it is a bit random.
Apparently it is not the await ... some useEffect within the collapsible gets all excited once we start refreshing the store...

Reply:
> Any clue why this is working without await ? I spent a good while without it having the promise returning right away ... From looking at the console you need the await otherwise you get spammed with calls to the backend ... it ends up falling back on its feet but i think it is a bit random.

It might be because you used curly braces which returns `void` by default unless you explicitly return a `Promise` value.

Reply:
`async (dataset) => {dispatch(get(dataset.datasetID))}`

Without an the await keyword, it just returns void.